copyGcno.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/sh
```

Comment:
Please align the header with other shell files we have in `build-scripts` directory